### DISCUSSION TITLE
Why the hightlight color is wierd?

### DISCUSSION THREAD

**Discussion by MKrbm** [nvim-tree/nvim-tree.lua#2780] [General]
I'm digging around the highlight setting to change nvimtree color. 
But found no highlight group changes the color below the files as you can see in the picture. 
How to make entire nvimtree pane red (for example) ?
<img width="569" alt="Screenshot 0006-05-22 at 11 21 53" src="https://github.com/nvim-tree/nvim-tree.lua/assets/57752203/34f57433-f37a-4f26-89f0-7845968e67b3">

**Comment by alex-courtis** [comment]
See `:help nvim-tree-highlight`

I think you might be after `NvimTreeNormal` e.g.
`:hi NvimTreeNormal guibg=red`

**Reply by MKrbm** [reply]
I already set :hi NvimTreeNormal guibg=red, which is done in the setting on the right side of ss.
However, the color doesn't apply entirely on the left window.

**Reply by alex-courtis** [reply]
You can see _all_ the nvim highlights via `:NvimTreeHiTest`

There are three highlight groups that affect that area: `NvimTreeEndOfBuffer`, `NonText` and `EndOfBuffer`.

`NvimTreeEndOfBuffer` links to `EndOfBuffer` by defaults. nvim links `EndOfBuffer` to `NonText` by default.
